### PR TITLE
Newcomers_Guide.rst: Add reference to GitLab repo

### PR DESCRIPTION
**Pull request by amisha2016**
It contains the documentation for Gitlab coala.

Closes https://github.com/coala/coala/issues/3901

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by amisha2016** [comment]
One can view it at [https://amisha.gdy.club/coala/docs/_build/html/Developers/Newcomers_Guide.html#optional-get-help-with-git](url)

**Comment by vibhanshuvaibhav** [comment]
Your shortlog should be in imperative present tense Use `Add` in place of `Added`. See this guide: https://api.coala.io/en/latest/Developers/Writing_Good_Commits.html
(jump to section Short Description under heading Shortlog)

Also, Please squash your 3 commits into 1. Follow this guide if you need help(jump to section Squashing your commits): http://api.coala.io/en/latest/Developers/Git_Basics.html

**Comment by vibhanshuvaibhav** [comment]
Squash your commits into a single commit. That will make things easy for you.

File: docs/Developers/Newcomers_Guide.rst [hunk 1]
```diff
@@ -164,6 +164,34 @@ git/GitHub, we strongly recommend following a tutorial, such as `this one
...
 will help you learn the basics.
 
+.. note::
+
+    coala also has some of its repositories on
+    `Gitlab <https://gitlab.com/coala>`_.
+    Gitlab contributions also count for the promotion of
+    newcomer to developer process.
+
+Why Gitlab?
```
Comment (ishanSrt) [inline docs/Developers/Newcomers_Guide.rst]: why this section is created? I don't see the issue mentions anything about explaining the importance of gitlab

File: docs/Developers/Newcomers_Guide.rst [hunk 2]
```diff
@@ -164,6 +164,34 @@ git/GitHub, we strongly recommend following a tutorial, such as `this one
...
+
+ - ``Authentication levels`` take this security a step further,
+   allowing users to give people access beyond a read/write level.
+   For example, you can give a team member access to issue tracking
+   without having to give them access to the code itself.
+
+We also have a
+`wiki page <https://github.com/coala/coala/wiki/GitLab>`_
+that will illustrate about Gitlab more.
+
```
Comment (ishanSrt) [inline docs/Developers/Newcomers_Guide.rst]: lines 191-194 are fine but the rest in the section `Why Gitlab?` seems redundant

**Comment by amisha2016** [comment]
@ishanSrt As per your suggestions I have made the changes. Please review it.:)

File: docs/Developers/Newcomers_Guide.rst [hunk 3]
```diff
@@ -164,6 +164,17 @@ git/GitHub, we strongly recommend following a tutorial, such as `this one
 We also have a `page dedicated to git commands <http://coala.io/git>`_ that
 will help you learn the basics.
 
+.. note::
+
+    coala also has some of its repositories on
```
Comment (ishanSrt) [inline docs/Developers/Newcomers_Guide.rst]: better preserve the style, that is already used. So frame this like `We also have some of our repositories.....`

Comment (ishanSrt) [reply docs/Developers/Newcomers_Guide.rst]: anyway coala always starting with a lower case is creating ambiguity on the sentence start

File: docs/Developers/Newcomers_Guide.rst [hunk 4]
```diff
@@ -164,6 +164,17 @@ git/GitHub, we strongly recommend following a tutorial, such as `this one
...
+    `Gitlab <https://gitlab.com/coala>`_.
+    Gitlab contributions also count for the promotion of
+    newcomer to developer process.
+
+We also have a
+`wiki page <https://github.com/coala/coala/wiki/GitLab>`_
+that will illustrate about Gitlab more.
```
Comment (ishanSrt) [inline docs/Developers/Newcomers_Guide.rst]: more about Gitlab

File: docs/Developers/Newcomers_Guide.rst [hunk 5]
```diff
@@ -164,6 +164,17 @@ git/GitHub, we strongly recommend following a tutorial, such as `this one
 We also have a `page dedicated to git commands <http://coala.io/git>`_ that
 will help you learn the basics.
 
+.. note::
+
+    coala also has some of its repositories on
+    `Gitlab <https://gitlab.com/coala>`_.
+    Gitlab contributions also count for the promotion of
+    newcomer to developer process.
```
Comment (ishanSrt) [inline docs/Developers/Newcomers_Guide.rst]: a newcomer to developer ~~process~~.

**Comment by amisha2016** [comment]
@ishanSrt Changes committed.
BTW, why coala starting with a lower case creates ambiguity on the sentence start.

File: docs/Developers/Newcomers_Guide.rst [hunk 6]
```diff
@@ -164,6 +164,17 @@ git/GitHub, we strongly recommend following a tutorial, such as `this one
 We also have a `page dedicated to git commands <http://coala.io/git>`_ that
 will help you learn the basics.
 
+.. note::
+
+    We also have some of our repositories on
+    `Gitlab <https://gitlab.com/coala>`_.
+    Gitlab contributions also count for the promotion of
+    newcomer to developer.
```
Comment (ishanSrt) [inline docs/Developers/Newcomers_Guide.rst]: "**a**" newcomer and add a line break in the body as the first line exceeds 70 characters.

Comment (ishanSrt) [reply docs/Developers/Newcomers_Guide.rst]: sorry the gitmate comments were for earlier commits

**Comment by ishanSrt** [comment]
its just that it was start of a sentence but you can't start coala with a capital letter. We should keep the docs grammatically correct 👍 and unambiguous.

File: docs/Developers/Newcomers_Guide.rst (hunk 6 above)
Comment (amisha2016) [reply docs/Developers/Newcomers_Guide.rst]: Sorry, I did not understand your comment. Is this PR correct now?

Comment (ishanSrt) [reply docs/Developers/Newcomers_Guide.rst]: just add "a" before newcomer and rest is fine

Comment (ishanSrt) [reply docs/Developers/Newcomers_Guide.rst]: commit body mention~~s~~

File: docs/Developers/Newcomers_Guide.rst [hunk 7]
```diff
@@ -164,6 +164,17 @@ git/GitHub, we strongly recommend following a tutorial, such as `this one
 We also have a `page dedicated to git commands <http://coala.io/git>`_ that
 will help you learn the basics.
 
+.. note::
+
+    We also have some of our repositories on
+    `Gitlab <https://gitlab.com/coala>`_.
+    Gitlab contributions also count for the promotion of a
```
Comment (Makman2) [inline docs/Developers/Newcomers_Guide.rst]: shouldn't it be:

... count ~~for~~ **to** ?

File: docs/Developers/Newcomers_Guide.rst [hunk 8]
```diff
@@ -164,6 +164,17 @@ git/GitHub, we strongly recommend following a tutorial, such as `this one
 We also have a `page dedicated to git commands <http://coala.io/git>`_ that
 will help you learn the basics.
 
+.. note::
+
+    We also have some of our repositories on
```
Comment (Makman2) [inline docs/Developers/Newcomers_Guide.rst]: Too many `We also have`s ;) Also see below :+1:

File: docs/Developers/Newcomers_Guide.rst [hunk 9]
```diff
@@ -164,6 +164,17 @@ git/GitHub, we strongly recommend following a tutorial, such as `this one
...
+    `Gitlab <https://gitlab.com/coala>`_.
+    Gitlab contributions also count for the promotion of a
+    newcomer to developer.
+
+We also have a
+`wiki page <https://github.com/coala/coala/wiki/GitLab>`_
+that will illustrate more about Gitlab.
```
Comment (Makman2) [inline docs/Developers/Newcomers_Guide.rst]: This seems to kinda refer to the note, which is optional and doesn't count to the "main-text". If GitLab was only mentioned in a note, you aren't allowed to reference it here, that's confusing.

Maybe this piece of text fits better if you move it upwards a bit?

**Comment by Makman2** [comment]
unack 35b2dcb498fc5eaf1cb1942c84360b51e76878c9

**Comment by amisha2016** [comment]
@Makman2 Please review it.:)

File: docs/Developers/Newcomers_Guide.rst [hunk 10]
```diff
@@ -164,6 +164,14 @@ git/GitHub, we strongly recommend following a tutorial, such as `this one
 We also have a `page dedicated to git commands <http://coala.io/git>`_ that
 will help you learn the basics.
 
+.. note::
+
+    You can view some of our repositories on
+    `Gitlab <https://gitlab.com/coala>`_.
```
Comment (Makman2) [inline docs/Developers/Newcomers_Guide.rst]: Git**L**ab ;)

File: docs/Developers/Newcomers_Guide.rst [hunk 11]
```diff
@@ -164,6 +164,14 @@ git/GitHub, we strongly recommend following a tutorial, such as `this one
...
+    Gitlab contributions also count to the promotion of a
```
Comment (Makman2) [inline docs/Developers/Newcomers_Guide.rst]: I think the promotion aspect fits more at the beginning where the promotion is first mentioned.

**Comment by Makman2** [comment]
unack d1fbe9aaa6374f16dfd8b48d37bbfd9bc821e793

**Comment by amisha2016** [comment]
@Makman2 Is it correct now?

File: docs/Developers/Newcomers_Guide.rst [hunk 12]
```diff
@@ -164,6 +165,13 @@ git/GitHub, we strongly recommend following a tutorial, such as `this one
 We also have a `page dedicated to git commands <http://coala.io/git>`_ that
 will help you learn the basics.
 
+.. note::
+
+    You can view some of our repositories on
+    `GitLab <https://gitlab.com/coala>`_.
+    For more information about it, browse to our
+    `wiki page <https://github.com/coala/coala/wiki/GitLab>`_.
```
Comment (Makman2) [inline docs/Developers/Newcomers_Guide.rst]: I think it should go down one paragraph to "pick an issue" :) That fits better, so people have more options to choose issues from :)

**Comment by Makman2** [comment]
Sorry for taking so long. I would just move the note to a different paragraph, then it's good to go :)

**Comment by Makman2** [comment]
unack fcb61ad0087dc06392c08ed441780e25c45e04d2

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala/commit/fcb61ad0087dc06392c08ed441780e25c45e04d2

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala/pull/5088/commits/77481502ee98916f99432a954f78df20c62d8632

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**